### PR TITLE
redirects: Redirect also to correct instance

### PR DESCRIPTION
**Pull request by kulla**
Fixes https://github.com/serlo/serlo.org-cloudflare-worker/issues/70

### CODE REVIEW

File: src/utils/index.tsx [hunk 1]
```diff
@@ -98,10 +102,17 @@ export async function getPathInfo(
...
+          alias
+          instance
+        }
+        ... on AbstractRepository {
+          alias
+          instance
+        }
+        ... on User {
+          alias
+        }
```
Comment (CarolinJaser) [inline src/utils/index.tsx:115]: Why is AbstractEntity left out here?

Comment (kulla) [reply src/utils/index.tsx:115]: Because every type implementing AbstractEntity also implements AbstractRepository

File: src/utils/index.tsx [hunk 2]
```diff
@@ -57,7 +57,10 @@ export function getCookieValue(
         .map((c) => c.substring(name.length + 1))[0] ?? null
 }
 
-const PathInfo = t.type({ typename: t.string, currentPath: t.string })
+const PathInfo = t.intersection([
+  t.type({ typename: t.string, currentPath: t.string }),
+  t.partial({ instance: t.string }),
+])
```
Comment (CarolinJaser) [inline src/utils/index.tsx:63]: Could you explain the logic behind this?

Comment (kulla) [reply src/utils/index.tsx:63]: Yes, let's use the call later :smile:

File: __tests__/__utils__/services/database.ts [hunk 3]
```diff
@@ -39,4 +39,5 @@ interface AbstractUuid<Typename extends string> {
   id?: number
   alias?: string
   oldAlias?: string
+  instance?: 'de' | 'en' | 'ta'
```
Comment (inyono) [inline __tests__/__utils__/services/database.ts:42]: We already have an Instance type for that: https://github.com/serlo/serlo.org-cloudflare-worker/blob/03cf8e731eebb97c41b1458b75c04921868d820c/src/utils/index.tsx#L34

**Review by CarolinJaser — APPROVED** [review]

**Pull request merged**